bool centerStatus

int32 centerX
int32 centerY
